File: Net/Controls/ToolbarControlZoomFactorExtension/VBNet/Extension/IZoomExtension.vb
Option Explicit On 

'IZoomExtension interface
Public Interface IZoomExtension
    Property ZoomFactor() As Double
End Interface

Public Class myIZoomExtension
    Implements IZoomExtension

    Public Property ZoomFactor() As Double Implements IZoomExtension.ZoomFactor
        Get
        End Get
        Set(ByVal Value As Double)
        End Set
    End Property
End Class

File: Net/Controls/ToolbarControlZoomFactorExtension/CSharp/Extension/IZoomExtension.cs
//IZoomExtension interface
public interface IZoomExtension
{
	double ZoomFactor
	{
		get;
		set;
	}
}

public class myIZoomExtension: IZoomExtension
{
	public double ZoomFactor 
	{
		get 
		{
			return 0;
		}

		set 
		{
		}
	}
}